sphinx_changelog/towncrier.py

```diff
@@ -57,17 +54,8 @@ def generate_changelog_for_docs(directory, skip_if_empty=True, underline=1):
 
     definitions = config.types
 
-    if config.directory:
-        base_directory = os.path.abspath(config.directory)
-        fragment_directory = None
-    else:
-        base_directory = os.path.abspath(
-            os.path.join(directory, config.package_dir, config.package)
-        )
-        fragment_directory = "newsfragments"
-
     fragments, fragment_filenames = find_fragments(
-        base_directory, config.sections, fragment_directory, definitions
+        base_directory, config, strict=False
```

Comment:
I've set `strict=False` here to ensure the same `sphinx-changelog` behaviour.

If we opt to set this to `True` then users may have to explicitly include their template file to the configured `ignored` files.